docs/developer-guide/base-maps/using-with-maplibre.md

```diff
@@ -0,0 +1,248 @@
+# Using with Maplibre
+
+| Pure JS | React | Overlaid | Interleaved |
+| ----- | ----- | ----- | ----- |
+|  ✓ | ✓ | [example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/maplibre) | [example](https://deck.gl/gallery/maplibre-overlay) |
+
+[Maplibre GL JS](https://maplibre.org/maplibre-gl-js/docs/) is a powerful open-source library that uses WebGL to render interactive maps from vector tiles in a browser. deck.gl's `MapView` can sync perfectly with the camera of Maplibre, at every zoom level and rotation angle.
```

Comment:
In mapbox there's a nice looking image in this part, can maplibre has a nice looking image too?

Reply:
Any suggestions? The Mapbox one contains building/style from the Mapbox data service, so I didn't want to reuse it. Anything new features worth showcasing of Maplibre?

Reply:
I looked at the deck.gl examples in maplibre, but they don't create any wow effect.
https://maplibre.org/maplibre-gl-js/docs/examples/toggle-deckgl-layer/
https://maplibre.org/maplibre-gl-js/docs/examples/add-deckgl-layer-using-rest-api/

But I think any picture is better than no picture, feel free to take a picture from any of the above examples.